test/remote-test.js

```diff
@@ -1873,3 +1960,499 @@ describe('Remote', function() {
     });
   });
 });
+
+describe.skip('Request API consistency tests', function() {
```

Comment:
Yeah, maybe just remove them and add them when they are ready (using fixtures or whatever you wanted to do)?